Patch from fedora.
Build of kdenetwork fails with:

        cp: cannot stat `./debian/tmp/usr/bin/kppp': No such file or directory

Which is due to:

        checking if kppp should be compiled... no

inclusion of if_ppp.h fails to get used, since aligned_u64 isn't defined. The following
patch from fedora appears to rectify building kppp with a recent linux-libc-dev.

diff -up kdenetwork-3.5.9/kppp/configure.in.in.ppp kdenetwork-3.5.9/kppp/configure.in.in
--- kdenetwork-3.5.9/kppp/configure.in.in.ppp   2008-02-13 03:37:48.000000000 -0600
+++ kdenetwork-3.5.9/kppp/configure.in.in       2008-02-15 21:29:00.000000000 -0600
@@ -30,6 +30,9 @@ AC_TRY_LINK([
   #ifndef STREAMS 
 
     #if defined(linux)
+      #ifndef aligned_u64
+        #define aligned_u64 unsigned long long __attribute__((aligned(8)))
+      #endif
       #include <linux/if_ppp.h>
     #elif defined(__DragonFly__)
       #include <net/ppp/if_ppp.h>
diff -up kdenetwork-3.5.9/kppp/opener.cpp.ppp kdenetwork-3.5.9/kppp/opener.cpp
--- kdenetwork-3.5.9/kppp/opener.cpp.ppp        2008-02-13 03:37:48.000000000 -0600
+++ kdenetwork-3.5.9/kppp/opener.cpp    2008-02-15 21:30:44.000000000 -0600
@@ -73,6 +73,9 @@ extern "C" int _Precvmsg(int, void*, int
 #    include <net/if.h>
 #    include <net/ppp/if_ppp.h>
 #  elif defined HAVE_LINUX_IF_PPP_H
+#    ifndef aligned_u64
+#      define aligned_u64 unsigned long long __attribute__((aligned(8)))
+#    endif
 #    include <linux/if_ppp.h>
 #  endif
 #else
diff -up kdenetwork-3.5.9/kppp/pppstats.cpp.ppp kdenetwork-3.5.9/kppp/pppstats.cpp
--- kdenetwork-3.5.9/kppp/pppstats.cpp.ppp      2008-02-13 03:37:48.000000000 -0600
+++ kdenetwork-3.5.9/kppp/pppstats.cpp  2008-02-15 21:29:00.000000000 -0600
@@ -73,6 +73,9 @@
  #ifndef HAVE_NET_IF_PPP_H
   #ifdef HAVE_LINUX_IF_PPP_H
    #include <linux/if.h>
+   #ifndef aligned_u64
+    #define aligned_u64 unsigned long long __attribute__((aligned(8)))
+   #endif
    #include <linux/if_ppp.h>
   #elif defined(__DragonFly__)
    #include <net/if.h>

